README.md

```diff
@@ -1 +1,105 @@
-# angular-testing-filters-readme
\ No newline at end of file
+# Testing Filters
+
+## Overview
+
+Now that we can utilise the power of filters, let's test them to make sure they're returning the correct data.
+
+## Objectives
+
+- Describe filter testing
+- Write a unit test for filters
+
+## Setup tests
+
+As we learned earlier, we can use either filters in the DOM or in our controllers.
+
+We're going to be writing our filter's tests using Karma and Jasmine - this means we will have to be using the filters in our controllers as we cannot test our DOM.
```

Comment:
Can you add more context about why we do the tests this way? Why is this the preferred way to test filters? Is it because we get better performance from creating our filters in the controllers, or simply because it's too difficult or impossible to test the output to the DOM?